cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -403,13 +411,141 @@ include::../indexes/drop-an-index.asciidoc[leveloffset=+1]
 include::../indexes/drop-a-non-existing-index.asciidoc[leveloffset=+1]
 
 
+// TODO: should this section live here? or at some other level in the documentation
+[[indexes-future-indexes]]
+== Future indexes
+
+There are preparations for introducing two new types of indexes, point and range indexes.
```

Comment:
Should we say something here about how usable they are and when they are coming or should this section just say you can read more about it here `<link-to-ops-manual>`?

Reply:
If we want to have examples of using the indexes similar to the query tuning section here as well I think we should add something but if not I think a link is sufficient

(I guess planner team would be responsible for such examples)

Reply:
Yes I also think it make sense to add a sentence in the beginning about that they aren't used yet